pkg/dockercreds/k8sdockercreds/k8s_keychain.go

```diff
@@ -34,11 +34,11 @@ func NewSecretKeychainFactory(client k8sclient.Interface) (*k8sSecretKeychainFac
 
 func (f *k8sSecretKeychainFactory) KeychainForSecretRef(ctx context.Context, ref registry.SecretRef) (authn.Keychain, error) {
 	if !ref.IsNamespaced() {
-		k8sNodeKeychain, err := k8schain.NewNoClient(context.Background())
+		k8sKeychain, err := k8schain.NewNoClient(context.Background())
```

Comment:
Nice.

Reply:
I gave it the weird name in the first place